libs/sdk-backend-tiger/src/backend/index.ts

```diff
@@ -193,8 +308,312 @@ export class TigerBackend implements IAnalyticalBackend {
                         return "";
                     }
                 },
+                getOrganizationAllowedOrigins: async (organizationId: string) => {
+                    try {
+                        return await this.authApiCall(async (sdk) => {
+                            const result = await sdk.entities.getEntityOrganizations({ id: organizationId });
+                            return result.data?.data?.attributes?.allowedOrigins || [];
+                        });
+                    } catch (error) {
+                        return [];
+                    }
+                },
+                getOrganizationPermissions: async (organizationId: string) => {
+                    return await this.authApiCall(async (sdk) => {
+                        try {
```

Comment:
Should the style be unified? `getOrganizationAllowedOrigins` has try catch on top of authApiCall, `getOrganizationPermissions` has it inverted. I am not sure it is intentional, if not we should pick one approach and stick with it.